pkg/virtctl/config/config.go

```diff
@@ -0,0 +1,137 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+	"kubevirt.io/client-go/log"
+)
+
+func NewCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
```

Comment:
This command is not wired up in the root cmd.

Reply:
Done